<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>End</class>
 <widget class="QDialog" name="End">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Over</string>
  </property>
  <widget class="QLabel" name="winnerText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>82</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>WINNER: PLAYER</string>
   </property>
  </widget>
  <widget class="QLabel" name="winnerNumber">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>73</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>180</y>
     <width>146</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttonLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>GAME OVER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitBtn">
      <property name="text">
       <string>Quit</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replayBtn">
      <property name="text">
       <string>Replay</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
